Microsoft.Toolkit.Uwp/Helpers/SystemInformation.cs

```diff
@@ -48,6 +48,30 @@ public static class SystemInformation
 
         /// <summary>
         /// Gets device's family
+        /// <para></para>
+        /// Common values include:
+        /// <list type="bullet">
+        /// <item>
+        /// <term>"Windows.Desktop"</term>
+        /// </item>
+        /// <item>
+        /// <term>"Windows.Mobile"</term>
+        /// </item>
+        /// <item>
+        /// <term>"Windows.Xbox"</term>
+        /// </item>
+        /// <item>
+        /// <term>"Windows.Holographic"</term>
```

Comment:
Should we add more after each point like common devices that use these types?  i.e. HoloLens and Mixed Reality devices for holographic and Surface Hub for Team.

Reply:
Yes. We can add more information on the `<description>` node. The best option would be to add it on each item value but we need to be sure what to add on it. Like for `Windows.Desktop`, I suppose it englobes desktops, laptops and tablets.

Reply:
We can add it in a future PR if it is necessary.